spec/requests/portfolios_rbac_spec.rb

```diff
@@ -7,7 +7,7 @@
   let(:block_access_obj) { instance_double(RBAC::Access, :accessible? => false) }
 
   describe "GET /portfolios" do
-    it 'returns status code 200' do
+    xit 'returns status code 200' do
```

Comment:
Do you mean 'pending'? I didn't know about `xit`.

Reply:
xit is the same as pending